scoap3/utils/throttling.py

```diff
@@ -0,0 +1,39 @@
+from rest_framework.throttling import SimpleRateThrottle
+
+
+class UserGroupRateThrottle(SimpleRateThrottle):
```

Comment:
maybe we can implement something like this https://github.com/encode/django-rest-framework/blob/master/rest_framework/throttling.py#L205

and override the `get_rate` method, so it can check the user groups once and determine the rate

this way we register only one class (runs only one time in every request) and we also avoid multiple requests to get the user groups (with multiple classes and `get_cache_key` methods)